connectd/connectd.c

```diff
@@ -874,8 +874,10 @@ static struct io_plan *conn_init(struct io_conn *conn,
 			      "Can't connect to forproxy address");
 		break;
 	case ADDR_INTERNAL_WIREADDR:
+#if EXPERIMENTAL_FEATURES /* BOLT7 DNS RFC #911 */
 		/* DNS should have been resolved before */
 		assert(addr->u.wireaddr.type != ADDR_TYPE_DNS);
+#endif
```

Comment:
Don't need #ifdef around this, right?

Reply:
Sure, since we have the enum type `ADDR_TYPE_DNS` this precompiler `#if` is pretty pointless.